.prettierrc.json

```diff
@@ -1,3 +1,30 @@
 {
-  "parser": "html"
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "useTabs": true,
```

Comment:
Please set "useTabs" to `false` (https://google.github.io/styleguide/tsguide.html#whitespace-characters)

Reply:
Done